core/templates/dev/head/components/forms/custom-forms-directives/thumbnail-uploader.directive.ts

```diff
@@ -0,0 +1,186 @@
+// Copyright 2016 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Directive for uploading images.
+ */
+
+import Cropper from 'cropperjs';
```

Comment:
Can't `require` be used here?

Reply:
I think that it is okay to use `import` here, since we will be using msotly `import`s in Angular 8 anyway.

Reply:
Right

Reply:
OK, in that case I'll leave this as is. Thanks!

Reply:
Yes